src/index.ts

```diff
@@ -10,20 +13,28 @@ const dotsimus = new Client({
   ],
 });
 
-if (!process.env.DISCORD_TOKEN) {
-  throw new Error("Discord token not provided.");
+if (!process.env.NODE_ENV) {
```

Comment:
Instead of using FormatString, you can either use ternaries in a template literal (preferred) or use a tag function